sdk/test/trace/tracer_test.cc

```diff
@@ -379,3 +392,28 @@ TEST(Tracer, TestParentOrElseSampler)
   span_parent_off_2->End();
   ASSERT_EQ(0, spans_received_parent_off->size());
 }
+
+TEST(Tracer, TestRuntimeContextSpan)
+{
+
+  std::shared_ptr<std::vector<std::unique_ptr<SpanData>>> spans_received(
+      new std::vector<std::unique_ptr<SpanData>>);
+  auto tracer = initTracer(spans_received);
+
+  auto span_first  = tracer->StartSpan("span 1");
+  auto span_second = tracer->StartSpan("span 2");
+
+  ASSERT_EQ(0, spans_received->size());
```

Comment:
I always suggest using ASSERT_ only to prevent the test from crashing later (e.g. a pointer isn't null) and otherwise stick with EXPECT_    (IDK if there's a different expectation in OTel, or if the person that first wrote this test file was accustomed to a test framework that didn't have ASSERT & EXPECT as two different things).

If the test crashes, it kill prevent other test facets (TEST_F) from running, but generally we want to get as much information out of the test as possible, so e.g. here if all of the spans_received values are 1 greater than expected we know there's an extra span somewhere but the context based spans are probably working.

Reply:
Changed to EXPECT, but only for the test I added.